README.md

```diff
@@ -360,6 +366,33 @@ The following settings are set for you automatically by [pluto](sources/api/) ba
 * `settings.kubernetes.node-ip`: The IPv4 address of this node.
 * `settings.kubernetes.pod-infra-container-image`: The URI of the "pause" container.
 
+#### Amazon ECS settings
+
+The following settings can be optionally specified in order to join an Amazon ECS cluster.
+You should [specify them in user data](#using-user-data) as changes after your container instance has joined the cluster will not work.
+See the [setup guide](QUICKSTART-ECS.md) for *much* more detail on setting up Bottlerocket and ECS.
+* `settings.ecs.cluster`: The name or [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of your Amazon ECS cluster.
+  If left unspecified, Bottlerocket will join your `default` cluster.
+* `settings.ecs.instance-attributes`: [Attributes](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-placement-constraints.html#attributes) in the form of key, value pairs added when registering the container instance in the cluster.
+  * Example user data for setting up attributes:
+    ```
+    [settings.ecs.instance-attributes]
+    attribute1 = "foo"
+    attribute2 = "bar"
+    ```
+
+The following settings are optional and allow you to further configure your cluster.
+These settings can be changed at any time.
+* `settings.ecs.logging-drivers`: The logging drivers available on the container instance.
```

Comment:
```suggestion
* `settings.ecs.logging-drivers`: The list of logging drivers available on the container instance.
```
Small tweak to hint that it's a list type.  I don't think we need an explicit example, though.

Reply:
Done